libs/R-TypeLogic/Server/Systems/CollidableSystem.cpp

```diff
@@ -69,25 +85,50 @@ void Collide::collide(
                         sndLife.lifePoint = fstDamage.damagePoint;
                     sndLife.lifePoint -= fstDamage.damagePoint;
                     sndLife.modified = true;
+                    if (sndLife.lifePoint == 0 && fstEntity->contains<Player>() && sndEntity->contains<Enemy>()) {
```

Comment:
This functionality does not have anything to do here, it should be in death system or something similar

Reply:
:eyes: